Ubuntu 18.04 apt support

Currently the bintray repository doesn't support the Bionic Beaver LTS version.
Would you mind supporting this version?

Tried to see if Bintray now support Ubuntu Bionic but, after enabling ubuntu-18-04 and disabling every other target, running `compile:publishToBintray` I get:

```
Status: Downloaded newer image for lightbend-docker-registry.bintray.io/rp/reactive-cli-build-ubuntu-18-04:latest
docker: Error response from daemon: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"./command\": stat ./command: no such file or directory": unknown.
time="2018-08-14T16:35:57+01:00" level=error msg="error waiting for container: context canceled"
[debug] Forcing garbage collection...
[trace] Stack trace suppressed: run last root/compile:buildAll for the full output.
[error] (root/compile:buildAll) java.lang.AssertionError: assertion failed: Executing WrappedArray(docker, run, --rm=true, --env, BINTRAY_USER=none, --env, BINTRAY_PASS=none, -v, /d/reactive-cli/target/stage/ubuntu-18-04:/root/stage, -v, /d/reactive-cli/target/.ivy2/cache:/root/.ivy2/cache, -v, /d/reactive-cli/target/.sbt/launchers:/root/.sbt/launchers, lightbend-docker-registry.bintray.io/rp/reactive-cli-build-ubuntu-18-04:latest) yielded 127, expected 0
[error] Total time: 239 s, completed 14-Aug-2018 16:35:58
```

Same result:

```
> build ubuntu-17-04_17-10
[info] [ubuntu-17-04_17-10] building
[info] [ubuntu-17-04_17-10] copying project
latest: Pulling from rp/reactive-cli-build-ubuntu-17-04_17-10
8b23367590c3: Pulling fs layer
d18b5715fdd4: Pulling fs layer
5ba295b2a11b: Pulling fs layer
31ace8c1d433: Pulling fs layer
95434b428ba1: Pulling fs layer
26046d65c450: Pulling fs layer
8640e7d96a66: Pulling fs layer
31ace8c1d433: Waiting
95434b428ba1: Waiting
26046d65c450: Waiting
8640e7d96a66: Waiting
5ba295b2a11b: Download complete
d18b5715fdd4: Verifying Checksum
d18b5715fdd4: Download complete
95434b428ba1: Verifying Checksum
95434b428ba1: Download complete
31ace8c1d433: Verifying Checksum
31ace8c1d433: Download complete
8640e7d96a66: Verifying Checksum
8640e7d96a66: Download complete
8b23367590c3: Verifying Checksum
8b23367590c3: Download complete
8b23367590c3: Pull complete
d18b5715fdd4: Pull complete
5ba295b2a11b: Pull complete
31ace8c1d433: Pull complete
95434b428ba1: Pull complete
26046d65c450: Verifying Checksum
26046d65c450: Download complete
26046d65c450: Pull complete
8640e7d96a66: Pull complete
Digest: sha256:cb4d2b5466b4cc694da7b03cb2db4d4dfaa4745995933ddd7708593ad1f3802e
Status: Downloaded newer image for lightbend-docker-registry.bintray.io/rp/reactive-cli-build-ubuntu-17-04_17-10:latest
docker: Error response from daemon: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"./command\": stat ./command: no such file or directory": unknown.
[trace] Stack trace suppressed: run last root/compile:build for the full output.
[error] (root/compile:build) java.lang.AssertionError: assertion failed: Executing WrappedArray(docker, run, --rm=true, --env, BINTRAY_USER=none, --env, BINTRAY_PASS=none, -v, /d/reactive-cli/target/stage/ubuntu-17-04_17-10:/root/stage, -v, /d/reactive-cli/target/.ivy2/cache:/root/.ivy2/cache, -v, /d/reactive-cli/target/.sbt/launchers:/root/.sbt/launchers, lightbend-docker-registry.bintray.io/rp/reactive-cli-build-ubuntu-17-04_17-10:latest) yielded 127, expected 0
[error] Total time: 220 s, completed 14-Aug-2018 17:33:46
```

--- a/project/BuildInfo.scala
+++ b/project/BuildInfo.scala
@@ -5,6 +5,7 @@ import AdditionalIO._
 object BuildInfo {
   val Builds =
     Vector(
+
       MuslBuild.tgz("tgz"),
 
       MuslBuild.rpm("centos-6", "el6", "bash"),
@@ -94,9 +95,8 @@ object BuildInfo {
                       |  apt-get -y update && \\
                       |  apt-get -y install bc build-essential jq openjdk-8-jre-headless ca-certificates-java clang-3.8 libcurl4-openssl-dev libgc-dev libre2-dev libunwind8-dev sbt
                       |""".stripMargin,
-        target = DebBuildTarget(Seq("zesty", "artful"), "main", "bash,libre2-3,libunwind8,libcurl3", Seq.empty)))
+        target = DebBuildTarget(Seq("zesty", "artful"), "main", "bash,libre2-3,libunwind8,libcurl3", Seq.empty)),
 
-  /*
       BuildInfo(
         name = "ubuntu-18-04",
         baseImage = "ubuntu:18.04",
@@ -109,7 +109,7 @@ object BuildInfo {
                       |  apt-get -y install bc build-essential jq openjdk-8-jre-headless ca-certificates-java clang-3.9 libcurl4-openssl-dev libgc-dev libre2-dev libunwind8-dev sbt
                       |""".stripMargin,
         target = DebBuildTarget(Seq("bionic"), "main", "bash,libre2-4,libunwind8,libcurl4", Seq.empty)))
-  */
+
 
   private def initialize(root: File): Unit = {
     val ivyDir = root / "target" / ".ivy2" / "cache"

--- a/project/BuildTarget.scala
+++ b/project/BuildTarget.scala
@@ -203,7 +203,7 @@ case class DebBuildTarget(distributions: Seq[String], components: String, depend
           |
           |bash ./build
           |
-          |DPKGVER="$$(dpkg-query --showformat='$${Version}' -W dpkg)"
+          |DPKGVER="$$(dpkg --version | sed -e 's/^.*version \\([^ ]*\\) .*$$/\\1/;q')"
           |RECENTDPKG=0
           |dpkg --compare-versions "$$DPKGVER" "lt" "1.19" || RECENTDPKG=1
           |if [[ "$$RECENTDPKG" == "0" ]]

--- a/project/BuildTarget.scala
+++ b/project/BuildTarget.scala
@@ -203,7 +203,15 @@ case class DebBuildTarget(distributions: Seq[String], components: String, depend
           |
           |bash ./build
           |
-          |dpkg --build package
+          |DPKGVER="$$(dpkg-query --showformat='$${Version}' -W dpkg)"
+          |RECENTDPKG=0
+          |dpkg --compare-versions "$$DPKGVER" "lt" "1.19" || RECENTDPKG=1
+          |if [[ "$$RECENTDPKG" == "0" ]]
+          |then
+          |  dpkg-deb -b package
+          |else
+          |  dpkg-deb -b --no-uniform-compression package
+          |fi
           |
           |mv package.deb output/reactive-cli_$version-${distributions.mkString("-")}_$architecture.deb
           |""".stripMargin

